hypeme.py

```diff
@@ -19,147 +19,144 @@
+
 
 def removeDisallowedFilenameChars(filename):
-    cleanedFilename = unicodedata.normalize('NFKD', filename).encode('ASCII', 'ignore')
+    cleanedFilename = unicodedata.normalize('NFKD', filename).encode('ASCII',
+                                                                     'ignore')
     return ''.join(c for c in cleanedFilename if c in validFilenameChars)
 
 
 class HypeScraper:
-  
-  def __init__(self):
-    pass
-    
-  def start(self):
-    print "--------STARTING DOWNLOAD--------"
-    print "\tURL : {} ".format(HYPEM_URL)
-    print "\tPAGES: {}".format(NUMBER_OF_PAGES)
-    
-    for i in range(1, NUMBER_OF_PAGES + 1):
-    
-      print "PARSING PAGE: {}".format(i)
-    
-      page_url = HYPEM_URL + "/{}".format(i)
-      html, cookie = self.get_html_file(page_url)
-
-      if DEBUG:
-        html_file = open("hypeHTML.html", "w")
-        html_file.write(html)
-        html_file.close()
-        
-      tracks = self.parse_html(html)
-      
-      print "\tPARSED {} SONGS".format(len(tracks) )
-      
-      self.download_songs(tracks, cookie)
-      
-      
-  def get_html_file(self, url):
-    data = {'ax':1 ,
-              'ts': time()
-          }
-    data_encoded = urllib.urlencode(data)
-    complete_url = url + "?{}".format(data_encoded)
-    request = urllib2.Request(complete_url)
-    response = urllib2.urlopen(request)
-    #save our cookie
-    cookie = response.headers.get('Set-Cookie')
-    #grab the HTML
-    html = response.read()
-    response.close()
-    return html, cookie
-    
-  def parse_html(self, html):
-    track_list = []
-    soup = BeautifulSoup(html)
-    html_tracks = soup.find(id="displayList-data")
-    if html_tracks is None:
-      return track_list
-    try:
-      track_list = json.loads(html_tracks.text)
-      if DEBUG:
-        print json.dumps(track_list, sort_keys=True,indent=4, separators=(',', ': '))
-      return track_list[u'tracks']
-    except ValueError:
-      print "Hypemachine contained invalid JSON."
-      return track_list
-      
-  #tracks have id, title, artist, key
-  def download_songs(self, tracks, cookie):
-  
-    print "\tDOWNLOADING SONGS..."
-    for track in tracks:
-    
-      key = track[u"key"]
-      id = track[u"id"]
-      artist = removeDisallowedFilenameChars(track[u"artist"])
-      title = removeDisallowedFilenameChars(track[u"song"])
-      type = track[u"type"]
-      
-      print "\tFETCHING SONG...."
-      
-      print u"\t{} by {}".format(title, artist)
-      
-      if type is False:
-        print "\tSKIPPING SONG SINCE NO LONGER AVAILABLE..."
-        continue
-       
-      try:
-        serve_url = "http://hypem.com/serve/source/{}/{}".format(id, key)
-        request = urllib2.Request(serve_url, "" , {'Content-Type': 'application/json'})
-        request.add_header('cookie', cookie)
+
+    def __init__(self):
+        pass
+
+    def start(self):
+        print '--------STARTING DOWNLOAD--------'

```

Comment:
Yep. Replaced two spaces with four spaces for indentation (from [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation))